plugins/catalog-backend/src/database/CommonDatabase.ts

```diff
@@ -219,15 +219,16 @@ export class CommonDatabase implements Database {
 
     let entitiesQuery = tx<DbEntitiesRow>('entities');
 
-    for (const filter of filters || []) {
-      const key = filter.key.toLowerCase().replace(/[*]/g, '%');
-      const keyOp = filter.key.includes('*') ? 'like' : '=';
+    for (const [matchKey, matchVal] of Object.entries(filters ?? {})) {
+      const key = matchKey.toLowerCase().replace(/[*]/g, '%');
+      const keyOp = key.includes('*') ? 'like' : '=';
```

Comment:
Check for `'%'` instead here right? Both because it used to be `matchKey`, but also because I assume both `*` and `%` would be valid placeholders, unless we escape existing `%`'s

Reply:
Ah yeah good eye, should be matchKey. No escaping necessary, the op is equality instead of LIKE if there were no asterisks

Reply:
Thinking about the behaviour of `'abc%123*xyz`' vs `'abc%123%xyz'`

Reply:
Mmh ok, yeah I guess escaping would be best, I'll look into it

Reply:
Shit the escaping is db dependent and knex doesn't seem to offer help. How about we just make this good enough for now

Reply:
I'd say the least surprising behaviour would be to treat both `%` and `*` as wildcards?

Reply:
I really didn't want to (feels like it leaks db implementation just because) but sure it's the easiest now

Reply:
fix'd